config/secrets.yml

```diff
@@ -0,0 +1,2 @@
+development:
+  secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
```

Comment:
Please change `development` key to `shared` to enable `secret_key_base` in all environments